src/state/index.js

```diff
@@ -12,6 +12,7 @@ if (session && session.auth) {
 
 const defaultState = {
   showSidebar: false,
+  stack: {},
```

Comment:
can you maybe name it `stackToCopy` or something similar? To make it clear?
Also when landing on newstack and when you added it to the state, you could clear it again.